services/web-app/components/glossary/glossary.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import GlossaryList from './glossary-list'
+import GlossaryNav from './glossary-nav'
+
+const Glossary = () => {
+  const glossary = require('@/data/glossary')
+  return (
+    <div className="glossary">
+      <GlossaryNav />
+      <div className="cds--row">
+        <div className="cds--col-lg-7">
+          <GlossaryList glossary={glossary} />
+        </div>
+        <div className="cds--col-lg-1" />
+      </div>
+    </div>
```

Comment:
```suggestion
    <Grid>
      <Column sm={4} md={8} lg={7}>
        <div className="glossary">
          <GlossaryNav />
          <GlossaryList glossary={glossary} />
        </div>
      </Column>
    </Grid>
```

Let's replace with updated grid components.